common/src/primitives/amount.rs

```diff
@@ -12,6 +12,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
```

Comment:
A very tiny nitpick: I would remove "//" from this line because otherwise the following comment looks like a part of the license header.

Reply:
Sounds good 👍